apps/condo/domains/common/components/Alert.tsx

```diff
@@ -0,0 +1,54 @@
+import styled from '@emotion/styled'
+import { Alert as DefaultAlert, AlertProps } from 'antd'
+import React, { ComponentProps } from 'react'
+
+import { colors } from '@condo/domains/common/constants/style'
+
+const getAlertColorsByType = (type: ComponentProps<typeof Alert>['type']) => {
+    const defaultColors = {
+        bg: colors.infoAlertBg,
+        message: colors.infoAlertMessage,
+        description: colors.infoAlertDescription,
+    }
+
+    switch (type) {
```

Comment:
Looks strange. This function always return constant. So you don't need to use switch here

Reply:
it's better to add a TODO for that